spikesorters/basesorter.py

```diff
@@ -35,6 +35,9 @@ class BaseSorter:
     _gui_params = [
         {'name': 'output_folder', 'type': 'str', 'value':None, 'default':None,  'title': "Sorting output folder path"},
         {'name': 'grouping_property', 'type': 'str', 'value':None, 'default':None,  'title': "Will sort the recording by the given property"},
+        {'name': 'parallel', 'type': 'bool', 'value':False, 'default':False,  'title': "If the recording is sorted by a property, then it will do this in parallel"},
+        {'name': 'delete_output_folder', 'type': 'bool', 'value':False, 'default':False, 'title': "If True, delete the results of the sorter, otherwise, it won't."}
+        {'name': 'export_to_phy', 'type': 'bool', 'title': "If True, will save out results to Phy, otherwise, it won't save."}
```

Comment:
Hi @colehurwitz.
This last parameters is strange because this is not part of basesorter.

I guess taht you propose this option in spikely. In that case, why not adding this params directly in spikely ?

Reply:
@colehurwitz this is actually a good option! Why not appending the param directly in spikely?
Actually the same could be done for the probe file..

Reply:
Hi @samuelgarcia , so spikely currently displays all the parameters for a sorter using the gui_params in spikeinterface, it does not display any spikely specific params. Do you think it would be better to add this directly to spikely? 

I do something similar with probe_path in the spikeextractors gui params where I put probe path as a gui param in each recording extractor. What is nice about this is that some extractors need a probe path (for locations) and others don't so I can make some optional and some mandatory.

Reply:
Ok, I will remove these types of params and add them to spikely. Need to make a few pull requests across repos then